front/src/components/App/SalesArea/SalesArea.tsx

```diff
@@ -2,37 +2,85 @@ import Button from "../../UI/Button/Button";
 import Header from "../../UI/Header/Header";
 import RouteLink from "../../UI/RouteLink/RouteLink.styled";
 import { APP_PAGE } from "../App";
-import allegro_image from "../../Images/allegro.jpg";
-import olx_image from "../../Images/olx.jpg";
-import ebay_image from "../../Images/ebay.jpg";
-import CompaniesSector from "../../UI/CompanyView/CompaniesSector";
+import sony_image from "../../Images/sony.jpg";
+import jbl_image from "../../Images/jbl.jpg";
+import steelseries_image from "../../Images/myszka.jpg";
+import Products from "../../UI/ProductView/ProductSector";
 import { SalesAreaStyled } from "./SalesArea.styled";
+import Auctions from "../../UI/AuctionView/AuctionSector";
+import React, { useState } from "react";
+import ProductFilter from "../../UI/ProductView/ProductFilter";
 
-const EXAMPLES_COMPANIES = [
+enum InitState {
+  name = "Products",
+}
+
+const EXAMPLES_PRODUCTS = [
   {
     key: "1",
-    title: "Allegro",
-    img_src: allegro_image,
+    title: "TV Sony",
+    img_src: sony_image,
   },
   {
     key: "2",
-    title: "OLX",
-    img_src: olx_image,
+    title: "Słuchawki JBL",
+    img_src: jbl_image,
   },
   {
     key: "3",
-    title: "eBay",
-    img_src: ebay_image,
+    title: "Mysz Steelseries",
+    img_src: steelseries_image,
+  },
+];
+
+const EXAMPLES_AUCTIONS = [
+  {
+    key: "1",
+    site_id: "Allegro",
+    title: "TV Sony",
+    img_src: sony_image,
+  },
+  {
+    key: "2",
+    site_id: "Allegro",
+    title: "Słuchawki JBL",
+    img_src: jbl_image,
+  },
+  {
+    key: "3",
+    site_id: "Allegro",
+    title: "Mysz Steelseries",
+    img_src: steelseries_image,
+  },
+  {
+    key: "4",
+    site_id: "OLX",
+    title: "Mysz Steelseries",
+    img_src: steelseries_image,
   },
 ];
 
 const SalesArea = () => {
+  const [value, setValue] = useState<string>(InitState.name);
+  const [productsExamples, setProductsExamples] = useState(EXAMPLES_PRODUCTS);
+  const [auctionsExamples, setAuctionsExamples] = useState(EXAMPLES_AUCTIONS);
+
+  const handleChange = (selected: string) => {
+    setValue(selected);
+  };
+
+  const shouldDisplayProducts = value === "Products" || value === "Both";
```

Comment:
Wszystkie te stringi "Products", "Both" itd powinny zostać wyniesione do enuma. Dzięki temu bedziemy mogli zarządzać ich ilością i rodzajem z 1 miejsca. Pozwoli to uniknąć problemów w przyszłości gdy na przykład w innym pliku trzeba będzie użyć takiego samego rozróżnienia (później aktualizacja "gołych" stringów w kilku miejscach moze byc kłopotliwa dla programisty -  a gdy o jakimś zapomni to pojawi się bug w programie)